### PR TITLE
ENG-5142: Moves distributed workloads info into separate guide

### PR DESCRIPTION
**Pull request by bredamc**
ENG-5142: Moves distributed workloads info into separate guide

### CODE REVIEW

File: modules/configuring-the-distributed-workloads-components.adoc [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+.Prerequisites
+ifdef::upstream,self-managed[]
+* You have logged in to {openshift-platform} with the `cluster-admin` role.
+endif::[]
+ifdef::cloud-service[]
+* You have logged in to OpenShift with the `cluster-admin` role.
+endif::[]
+
+* You have access to the data science cluster.
```
Comment (MelissaFlinn) [inline modules/configuring-the-distributed-workloads-components.adoc:17]: combine first two bullet items into one:
 You have access to the cluster where ODH/RHOAI is installed.

File: modules/configuring-the-distributed-workloads-components.adoc [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+* You have installed {productname-long}.
+* You have sufficient resources. In addition to the base {productname-short} resources, you need 1.6 vCPU and 2 GiB memory to deploy the distributed workloads infrastructure.
```
Comment (MelissaFlinn) [inline modules/configuring-the-distributed-workloads-components.adoc:19]: SME question - how do they find out what the base resources are?

File: modules/configuring-the-distributed-workloads-components.adoc [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+* You have access to a Ray cluster image. For information about how to create a Ray cluster, see the link:https://docs.ray.io/en/latest/cluster/getting-started.html[Ray Clusters documentation].
++
+[NOTE]
+====
+Mutual Transport Layer Security (mTLS) is enabled by default in the CodeFlare component in {productname-short}.
+In the current {productname-short} version, `submissionMode=K8sJobMode` is not supported in the Ray job specification, so the KubeRay Operator cannot create a submitter Kubernetes Job to submit the Ray job.
+Instead, the Ray job specification must be configured to set `submissionMode=HTTPMode` only, so that the KubeRay Operator sends a request to the RayCluster to create a Ray job.
```
Comment (MelissaFlinn) [inline modules/configuring-the-distributed-workloads-components.adoc:26]: avoid passive voice "must be configured"

File: modules/configuring-the-distributed-workloads-components.adoc [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+====
+* You have access to the data sets and models to be used by the distributed workload.
```
Comment (MelissaFlinn) [inline modules/configuring-the-distributed-workloads-components.adoc:28]: You have access to the data sets and models that you want the distributed workload to use (or to process?)

File: modules/configuring-the-distributed-workloads-components.adoc [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+[source,bash]
+----
+/etc/pki/tls/certs/odh-ca-bundle.crt
+/etc/ssl/certs/odh-ca-bundle.crt
+----
+endif::[]
+ifdef::upstream[]
+* If you want to use self-signed certificates, you have added them to a central Certificate Authority (CA) bundle as described in link:{odhdocshome}/installing_open_data_hub/#understanding-certificates_certs[Understanding certificates in {productname-short}].
+No additional configuration is necessary to use those certificates with distributed workloads.
+The centrally configured self-signed certificates are automatically available in the workload pods at the following mount points:
```
Comment (MelissaFlinn) [inline modules/configuring-the-distributed-workloads-components.adoc:96]: centrally-configured

File: modules/configuring-the-distributed-workloads-components.adoc [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+[source,bash]
+----
+/etc/pki/tls/certs/odh-ca-bundle.crt
+/etc/ssl/certs/odh-ca-bundle.crt
+----
+endif::[]
+ifdef::self-managed[]
+* If you want to use self-signed certificates, you have added them to a central Certificate Authority (CA) bundle as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/working-with-certificates_certs[Working with certificates] (for disconnected environments, see link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}_in_a_disconnected_environment/working-with-certificates_certs[Working with certificates]).
+No additional configuration is necessary to use those certificates with distributed workloads.
+The centrally configured self-signed certificates are automatically available in the workload pods at the following mount points:
```
Comment (MelissaFlinn) [inline modules/configuring-the-distributed-workloads-components.adoc:77]: centrally-configured

**Review by MelissaFlinn — COMMENTED** [review]
just a few comments

File: modules/configuring-the-distributed-workloads-components.adoc (hunk 5 above)
Comment (bredamc) [reply modules/configuring-the-distributed-workloads-components.adoc:96]: Hyphens are not needed for -ly adverbs or adjectives (see IBM Style).

File: modules/configuring-the-distributed-workloads-components.adoc (hunk 6 above)
Comment (bredamc) [reply modules/configuring-the-distributed-workloads-components.adoc:77]: As above

File: modules/configuring-the-distributed-workloads-components.adoc (hunk 1 above)
Comment (bredamc) [reply modules/configuring-the-distributed-workloads-components.adoc:17]: I think we need to standardize this wording across all modules, I've added an item to our team agenda to discuss.

File: modules/configuring-the-distributed-workloads-components.adoc (hunk 2 above)
Comment (bredamc) [reply modules/configuring-the-distributed-workloads-components.adoc:19]: I've updated to link to the Installation Guide section.

**Pull request merged**